control/Director.php

```diff
@@ -422,13 +422,23 @@ public static function set_current_page($page) {
 	 */
 	public static function absoluteURL($url, $relativeToSiteBase = false) {
 		if(!isset($_SERVER['REQUEST_URI'])) return false;
-		
+
+		//a url of . or ./ is the same as an empty url
+/*
+		if ($url == '.' || $url == './') {
```

Comment:
I know, but single line conditionals are so horrible!

Also, I think they are against PSR and optional as part of our coding standard (emphasis added by myself)

> Statements with if **can** be written without braces on a single line as the block